pkg/storage/chunk/client/ibmcloud/cos_object_client.go

```diff
@@ -26,13 +28,16 @@ import (
 	"github.com/weaveworks/common/instrument"
 )
 
+const DEFAULT_COS_AUTH_ENDPOINT = "https://iam.cloud.ibm.com/identity/token"
```

Comment:
Is there any specific reason to capitalise the constant? Can't we follow the normal go variable convention here? Since in Go the things are exported if it's start with a capital letter. 
```suggestion
const defaultCOSAuthEndpoint = "https://iam.cloud.ibm.com/identity/token"
```

Reply:
The general convention is to capitalize the constants. Agree on the const being exported but don't have a strong argument whether it should be exported or not in this case.

Reply:
Let's be consistent with what is already in Loki.